website/docs/docs/deploy/continuous-integration.md

```diff
@@ -54,3 +55,7 @@ When you push a new commit to a PR, dbt Cloud enqueues a new CI run for the late
 ### Run slot treatment <Lifecycle status="team,enterprise" />
 
 CI runs don't consume run slots. This guarantees a CI check will never block a production run.
+
+### SQL linting <Lifecycle status="beta" />
+
+When enabled for your CI job, dbt invokes [SQLFluff](https://sqlfluff.com/) which is a modular and configurable SQL linter that warns you of complex functions, syntax, formatting, and compilation errors. It will lint all the SQL files in your project. If the linter runs into errors, you can specify dbt to fail the job or continue running it. 
```

Comment:
Not sure where exactly this belongs, but I think we need something about the .sqlfluff config file. Basically, if you don't have that file, we will lint your things using the default dbt Cloud linting settings. If you do have a sqlfluff in your project, then we will use the rules that are defined as we run linting. You can read more about it in general here https://docs.sqlfluff.com/en/stable/gettingstarted.html